config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-upgrade.yaml

```diff
@@ -32,6 +32,12 @@ periodics:
         requests:
           memory: "9000Mi"
           cpu: 2000m
+  annotations:
+    test_tab-name: kubeadm-kinder-upgrade-stable-master
+    testgrid-dashboards: sig-release-master-informing
```

Comment:
some of the `*kubeadm*` jobs are in multiple dashboards - e.g. release-master-informing and sig-cluster-lifecycle-kubeadm, but this change moves them only to sig-release-master-informing.

Reply:
You're right. I want to clarify that they're not missing; they just couldn't be migrated at the time.

Most of the `kubeadm` tabs set a custom `alert_stale_results_hours` setting, which couldn't be annotated; see the dashboard [sig-cluster-lifecycle-kubeadm](https://github.com/kubernetes/test-infra/blob/719fee7dafd21df0673abda9b30b43eee52bf766/testgrid/config.yaml#L2826). That's changed recently, so an improved version of this script will move those dashboard tabs, too.

Reply:
ok, i wanted to do this change myself and will send a PR today + some other updates, but if this PR merges before that i will rebase mine.